spec/routes/page_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Citygram::Routes::Page do
+  include Citygram::Routes::TestHelpers
+
+  describe 'GET /' do
+    context 'ROOT_CITY_TAG is set' do
+      before do
+        ENV['ROOT_CITY_TAG'] = 'new-york'
+      end
+
+      after do
+        ENV.delete 'ROOT_CITY_TAG'
+      end
+
+      it 'responds with 200 OK' do
+        get '/'
+        expect(last_response.status).to eq 200
+      end
+
+      it 'returns a non-empty body' do
+        get '/'
+        expect(last_response.body).not_to be_empty
+      end
+    end
+
+    context 'ROOT_CITY_TAG is not set' do
+      it 'raises an exception' do
+        expect do
+          get '/'
+        end.to raise_error Citygram::Models::City::NotFound
```

Comment:
Agreed, this still works for CodeForAmerica's instance as this class is loaded conditionally based on the presence of the environment variable.


https://github.com/codeforamerica/citygram/pull/182/files#diff-0a400c8217d53ff4f978163d7c61868cR10

That would only be that case if you navigated to the the `Citygram::Routes::Page` route. 

These routes are loaded conditionally based on the environment variable, so this is truly an edge case that I was only covering for completeness sake.

The alternative would be to chat this conditional to not fallback to nil and instead raise the missing environment variable error.